python/job_runner/reporting/file.py

```diff
@@ -0,0 +1,184 @@
+import json
+import os
+import shutil
+import socket
+import time
+
+from job_runner.io import cond_unlink
+from job_runner.reporting.message import Exited, Finish, Init, Running, Start
+
+
+class File(object):
+    LOG_file = "JOB_LOG"
+    ERROR_file = "ERROR"
+    EXIT_file = "EXIT"
+    STATUS_file = "STATUS"
+    OK_file = "OK"
+    STATUS_FILE = "status.json"
```

Comment:
couldnt jobs.json be defined here, instead of injected as magic strings everywhere?

Reply:
Also all the is "_file" except status which is "_FILE"

Reply:
I would maybe call STATUS_FILE STATUS_json, and perhaps fetch the values from their C counterpart in job_queue/forward_model.cpp

Reply:
> couldnt jobs.json be defined here, instead of injected as magic strings everywhere?
It could, but it is really a `JobRunner` thing.



> … perhaps fetch the values from their C counterpart in job_queue/forward_model.cpp
`#defines` are garbled compile time, so they're not in the DLL (at least not accessible using the original name).
And as Markus discuss in https://github.com/equinor/ert/issues/434#issuecomment-524188452 the real fix to this might be to drop the file system as means for communication between the queue and job_dispatch altogether?